### ISSUE TITLE
Remove "unnecessary" classes

### ISSUE THREAD

**Issue by vizotov** [yegor256/cactoos#915]
Based on the discussion in issue #913, we need to remove duplicate classes with the same name in different packages and leave only one class for each name.

In other classes, constructors that accept instances of removed classes as parameters must use instances of the remaining classes with the same name.

To replace the use of removed class constructors, it is necessary to implement or verify the existence of appropriate decorators and it's constructors. Decorators should have names like `ListOf`, `CollectionOf`, `TextOf`.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@vizotov[/z](https://www.0crat.com/u/vizotov) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C63314D6Z) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
The job [#915](https://github.com/yegor256/cactoos/issues/915) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C63314D6Z/1842346241); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#915` already assigned to @llorllale, can't assign to @llorllale

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @vizotov[/z](https://www.0crat.com/u/vizotov)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat waiting on outcome of #913

**Comment by 0crat** [comment]
> @0crat waiting on outcome of #913  ([here](https://github.com/yegor256/cactoos/issues/915#issuecomment-402332502))

@llorllale The impediment for [#915](https://github.com/yegor256/cactoos/issues/915) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The architect of the project has changed; @llorllale[/z](https://www.0crat.com/u/llorllale) is not at this role anymore; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) is the architect now

**Comment by victornoel** [comment]
@vizotov I think this is now the case, I'm going to close this because it is very old but please feel free to open a new ticket if needed

**Comment by 0crat** [comment]
Job [#915](https://github.com/yegor256/cactoos/issues/915) is not in the agenda of @llorllale[/z](https://www.0crat.com/u/llorllale), can't set title

**Comment by 0crat** [comment]
The project doesn't have enough funds, can't make a payment
